tests/integration/staking/keeper/determinstic_test.go

```diff
@@ -39,13 +39,13 @@ import (
 
 var (
 	validator1        = "cosmosvaloper1qqqryrs09ggeuqszqygqyqd2tgqmsqzewacjj7"
-	validatorAddr1, _ = sdk.ValAddressFromBech32(validator1)
+	validatorAddr1, _ = sdk.ValAddressFromHex(validator1)
 	validator2        = "cosmosvaloper1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj"
-	validatorAddr2, _ = sdk.ValAddressFromBech32(validator2)
+	validatorAddr2, _ = sdk.ValAddressFromHex(validator2)
```

Comment:
Same here and other places.

Reply:
As discussed, this will be taken care of after the migration...